src/storage/cloud-extensions.ts

```diff
@@ -0,0 +1,89 @@
+import {
+  Bucket,
+  BucketOptions,
+  File,
+  FileOptions,
+  Storage as StorageClient,
+} from "@google-cloud/storage";
+interface FirebaseMetadata {
+  name: string;
+  bucket: string;
+  generation: string;
+  metageneration: string;
+  contentType: string;
+  timeCreated: string;
+  updated: string;
+  storageClass: string;
+  size: string;
+  md5Hash: string;
+  contentEncoding: string;
+  contentDisposition: string;
+  crc32c: string;
+  etag: string;
+  downloadTokens: string;
+}
+export class FirebaseStorageFile extends File {
```

Comment:
please include jsdocs docs for all exported classes and methods

Reply:
Done.